tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Content/ContentWithDropdown.spec.ts

```diff
@@ -83,7 +83,7 @@ for (const dataTypeName of dataTypeNames) {
   });
 }
 
```

Comment:
Might not be needed, as we also have a test for 'can not publish a mandatory checkbox list with an empty value' with the release tag